framework_tvb/tvb/interfaces/web/controllers/simulator_controller.py

```diff
@@ -1030,6 +1031,23 @@ def load_burst_read_only(self, burst_config_id):
 
     @expose_fragment('simulator_fragment')
     def copy_simulator_configuration(self, burst_config_id):
+        form = self.get_first_fragment(burst_config_id)
+        common.add2session(common.KEY_BRANCH, False)
+        rendering_rules = SimulatorFragmentRenderingRules(form, SimulatorWizzardURLs.SET_CONNECTIVITY_URL,
+                                                          is_simulation_copy=True, is_simulation_readonly_load=True,
+                                                          is_first_fragment=True)
+        return rendering_rules.to_dict()
+
+    @expose_fragment('simulator_fragment')
+    def branch_simulator_configuration(self, burst_config_id):
+        form = self.get_first_fragment(burst_config_id)
+        common.add2session(common.KEY_BRANCH, True)
+        rendering_rules = SimulatorFragmentRenderingRules(form, SimulatorWizzardURLs.SET_CONNECTIVITY_URL,
+                                                          is_simulation_copy=True, is_simulation_readonly_load=True,
+                                                          is_first_fragment=True)
+        return rendering_rules.to_dict()
+
+    def get_first_fragment(self, burst_config_id):
```

Comment:
this can be private and maybe use a more specific name to what its intention is. Also, the line         `common.add2session(common.KEY_IS_SIMULATOR_COPY, True)` is not common for both copy&branch methods